fluentnamer/readme.md

```diff
@@ -19,17 +19,28 @@ pipeline:
     flatten-models: true
     flatten-payloads: true
     naming:
+      preserve-uppercase-max-length: 2
       override:
-        api: Api
-        cpk: Cpk
-        sas: Sas
-        url: Url
-        xml: Xml
+        ip: Ip
+        id: Id
   
   # allow developer to do transformations on the code model.
   modelerfour/new-transform:
     input: modelerfour
 
   fluentnamer:
     input: modelerfour/identity
+    naming:
+      override:
+        eTag: etag
+        userName: username
+        metaData: metadata
+        timeStamp: timestamp
+        hostName: hostname
+        webHook: webhook
+        coolDown: cooldown
+        resourceregion: resourceRegion
+        sTag: stag
+        tagname: tagName
+        tagvalue: tagValue
```

Comment:
Currently I hardcoded the list here. If this works well, we can merge a default list here, and a similar list configured in swagger `readme.java.md`.